### ISSUE TITLE
Return the Event(onClick) and the value in the onChange function

### ISSUE THREAD

**Issue by Den368** [yogaboll/react-switch#1]
Please return the value and the Event of the onClick funtion.

**Comment by yogaboll** [comment]
So if I understand you correctly, you want the onChange function to be given the event as a second argument in addition to the 'checked'-value so you can use it like this:

```
handleChange(checked, event) {
  // Do something with event.target or something
  this.setState({ checked });
}
  render() {
    return (
        <Switch
          onChange={this.handleChange}
          checked={this.state.checked}
        />
    );
  }
```
Is that correct?

**Comment by yogaboll** [comment]
@Den368 I implemented the above in the new 2.1.0 release on npm. Feel free to comment if it doesn't work for your use case.
